### ISSUE TITLE
Issue while compiling on version 3.0.1

### ISSUE THREAD

**Issue by hazzo** [ankurk91/vue-bootstrap-datetimepicker#15]
**I'm submitting a ...**  (check one with "x")
```
[X] Bug report => search github for a similar issue or PR before submitting
[ ] Feature request
[ ] Other, please describe
```

**Tell about your platform**
* eonasdan-bootstrap-datetimepicker version : 4.x.x
* Vue.js version : 2.x.x
* Browser name and version : Chrome|Firefox|Safari x.x.x
* This package version : 3.0.1

**Current behavior**

Error while compiling component and consequent error in build.

**Minimal reproduction of the problem with instructions**
![captura de pantalla 51](https://user-images.githubusercontent.com/13692810/30853104-27a33408-a2ae-11e7-9b0f-d4e75978d841.png)

**Comment by alrik11es** [comment]
Happened to me as well. Confirmed.

**Comment by ankurk91** [comment]
Rolled-back version 3.0.1 changes, should be fixed in v3.0.2.
Sorry for inconvenience guys.
